File: ayt-01/04DecToBin.c
#include <stdio.h>
typedef long long int uint64_t;
void bin(uint64_t n)
{
    if (n > 1) bin(n >> 1);
    printf("%llu ", n & 1);
}
int main(int argv, char **argc)
{
    bin(4);
    return 0;
}

File: ayt-01/04DecToBin.java
class printDecToBin
{ 
    static void printDecToBin(int n)
    {
        if (n > 1) bin(n >> 1);
        System.out.print(n & 1);
    }
    public static void main(String args[]) 
    { /* 100 */
        bin(4);
    } 
} 